### DISCUSSION TITLE
Unable to use AceEditor from react-ace library

### DISCUSSION THREAD

**Discussion by abhinavb05** [remix-run/remix#7653] [Q&A]
Hi,
I am unable to use AceEditor from react-ace library. Getting this error
`Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. `

**Comment by PuruVJ** [comment]
Same here. Anyone got the solution

**Comment by jjebennett** [comment]
Did anyone get a resolution to this?
